.readthedocs.yaml

```diff
@@ -0,0 +1,22 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.11"
+  apt_packages:
+    - graphviz
+
+sphinx:
+  configuration: docs/conf.py
+
+python:
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
```

Comment:
I didn't know about the `extra_requirements` option, so there is no separate `docs/requirements.txt` needed, by folding it into the main project's metadata. Thanks for learning this from your patch!